src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -437,16 +437,19 @@ public static RecipeBuilder<?> doParallelRecipes(Recipe currentRecipe, RecipeMap
 
         // Simulate the merging of the maximum amount of recipes
         // and limit by the amount we can successfully merge
+        // Note that limitByOutput will be less than or equal to multiplierByInputs after this, so Math.min does not need using
         int limitByOutput;
         limitByOutput = ParallelLogic.limitByOutputMerging(currentRecipe, exportInventory, exportFluids, multiplierByInputs, voidItems, voidFluids);
 
-        int limitByVoltage = Math.abs((int) (maxVoltage / currentRecipe.getEUt()));
-        int parallelizable = Math.min(limitByVoltage, Math.min(multiplierByInputs, limitByOutput));
-
-        if (parallelizable > 0) {
+        if (currentRecipe.getEUt() != 0) {
```

Comment:
Right now that you need to call getEUt() twice it would be wise to put it in local variable and use it from it.